DayrSuryan/DSEthiop12.xml

```diff
@@ -707,10 +730,13 @@ schematypens="http://relaxng.org/ns/structure/1.0"
 
                <history>
                 <origin>
-              <origDate notBefore="1920" notAfter="1950" evidence="lettering">Around the middle of the 20th century.</origDate>
+
+              <origDate notBefore="1920" notAfter="1953" evidence="lettering">Around the middle of the 20th century,
+                before 1954.</origDate>
                 The parts of the manuscript could be written by the same
                 scribe in Egypt, but at different times, trimmed and bound together
-                around 1950 (before 1954). The owner and possibly the scribe
+                around. The owner and possibly the scribe
```

Comment:
Now the sentence is incomplete, ending with "around."

Reply:
done